model/BaseDeliveryExecutionGetter.php

```diff
@@ -32,31 +34,27 @@
  */
 class BaseDeliveryExecutionGetter extends ConfigurableService implements DeliveryExecutionGetterInterface
 {
+    use OntologyAwareTrait;
+
     /**
      * Due to multiple implementation of DE storages it's difficult to check if DE exists
      * Ontology and KV storages allow us to check exists() but for other ones we have to try
      * to read mandatory 'status' property
      * @param string $deliveryExecutionId
      * @param LtiProvider $ltiProvider
-     * @return DeliveryExecutionInterface|null
      */
-    public function get($deliveryExecutionId, LtiProvider $ltiProvider)
+    public function get($deliveryExecutionId, LtiProvider $ltiProvider): ?DeliveryExecutionInterface
```

Comment:
We should update the interface and also `TenantDeliveryExecutionGetter` and other classes using this interface. Please, check the impact too.